client.go

```diff
@@ -65,7 +65,7 @@ func (client *GraphClient) Connect(username, password string) error {
 
 	resp, err := client.graph.Authenticate(username, password)
 	if err != nil {
-		log.Printf("Authentication fails, ErrorCode: %v, ErrorMsg: %s", resp.GetErrorCode(), resp.GetErrorMsg())
+		log.Printf("Authentication fails, %s", err.Error())
```

Comment:
When `resp` is not nil, it's better to print the error message of response. So we can add a check whether `resp == nil` here.

Reply:
I don't think this will happened follow the RPC interface behavior.